uk.co.compucorp.civicrm.tasksassignments/js/test/tasks-assignments/controllers/modal/modal-assignment.controller.spec.js

```diff
@@ -693,32 +831,47 @@ define([
           }
         }
       });
+      $rootScope.$digest();
     }
 
     /**
      * Selects the default client and case type assignations.
      */
     function selectDefaultClientAndAssignation () {
-      var selectedAssignment = $rootScope.cache.assignmentType.arr[0];
-
       scope.assignment.client_id = _.uniqueId();
       scope.assignment.case_type_id = selectedAssignment.id;
       scope.activity.activitySet = selectedAssignment.definition.activitySets[0];
     }
 
     /**
-     * Initializes the cache for assignment and document types.
+     * Initializes the cache for assignment, tasks, and document types.
```

Comment:
Please document why we are slicing arrays in half, this may be a bit difficult to understand without an explanation.